app/experimenter/experiments/serializers.py

```diff
@@ -49,6 +49,89 @@ class Meta:
         fields = ("code", "name")
 
 
+class ChangeLogSerializer(serializers.ModelSerializer):
+    variants = ExperimentVariantSerializer(many=True, required=False)
+    locales = LocaleSerializer(many=True, required=False)
+    countries = CountrySerializer(many=True, required=False)
+
+    class Meta:
+        model = Experiment
+        fields = (
```

Comment:
Experiment: the model with too many fields -_- I should've set this up differently, now it's gonna be a mess when we have to go in and split things up. *jared sticks a TECH DEBT sticker on his forehead*